test/e2e/manifest_test.go

```diff
@@ -44,19 +45,19 @@ var _ = Describe("Podman manifest", func() {
 		processTestResult(f)
 
 	})
-	It("podman manifest create", func() {
+	It("create w/o image", func() {
```

Comment:
I understand why you did this, but I will say having at least the manifest portion is a help when looking at failures.  If I only saw "create w/o image" as a test name, i would think containers ... anyways, your call

Reply:
On failure, the Description is included.